specification/dynatrace/resource-manager/readme.md

```diff
@@ -56,18 +79,25 @@ This is not used by Autorest itself.
 
 ```yaml $(swagger-to-sdk)
 swagger-to-sdk:
-  - repo: azure-sdk-for-python-track2
+  - repo: azure-sdk-for-python
   - repo: azure-sdk-for-java
   - repo: azure-sdk-for-go
   - repo: azure-sdk-for-js
-  - repo: azure-resource-manager-schemas
-  - repo: azure-cli-extensions
-  - repo: azure-powershell
+  - repo: azure-sdk-for-ruby
+    after_scripts:
+      - bundle install && rake arm:regen_all_profiles['azure_mgmt_dynatrace']
```

Comment:
I don't think we should add ruby and python is suppose to have a -track2 suffix. @msyyc